nuitka/plugins/standard/standard.nuitka-package.config.yml

```diff
@@ -7392,14 +7392,24 @@
       when: 'macos and version("zaber_motion") < (5,1,2)'
     - depends:
         - 'bindings'
-      when: 'version("zaber_motion") >= (5,1,2)'
+      when: 'version("zaber_motion") == (5,1,2)'
+    - depends:
+        - 'zaber_motion_bindings'
+      when: 'version("zaber_motion") >= (5,1,3)'
 
 - module-name: 'zaber_motion.bindings' # checksum: ebad5220
   anti-bloat:
     - description: 'normalize path for DLLs'
       replacements_plain:
         'os.path.join(os.path.dirname(__file__), "..", "bindings", lib_name)': 'os.path.normpath(os.path.join(os.path.dirname(__file__), "..", "bindings", lib_name))'
 
+- module-name: 'zaber_motion_bindings' # checksum: 679f33dc
+  dlls:
+    - from_filenames:
+        prefixes:
+          - 'zaber-motion-lib'
+      when: 'None is not version("zaber_motion") >= (5,1,3)'
```

Comment:
Ok, I see that works, and `bindings` was really the top level name, I now remember, but apparently they gave up on it quickly, I need to revert my stupid commit on factory.